react-native/react/native/pinentry.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+/* @flow */
+
+import engine from '../engine'
+import pinentryListeners from './pinentry-listeners'
+
+var initialized = false
+
+export function init () {
```

Comment:
that would change how the require above works, you wouldn't have `.init()` anymore.